R/mouse/mouse_deconvolution_methods.R

```diff
@@ -0,0 +1,228 @@
+#' Collection of deconvolution methods for mouse data.
+require(tidyverse)
+require(magrittr)
+require(rlang)
+require(ComICS)
+require(mMCPcounter)
+require(biomaRt)
+
+#' List of supported mouse deconvolution methods
+#'
+#' The methods currently supported are
+#' `mMcp_counter`, `seqimmucc`, `dcq`, `base`
+#'
+#' The object is a named vector. The names correspond to the display name of the method,
+#' the values to the internal name.
+#'
+#' @export
+deconvolution_methods_mouse = c("mMCPcounter"="mMcp_counter",
+                                "seqImmuCC"="seqimmucc", 
+                                "DCQ"="dcq", 
+                                "BASE"="base")
+
+
+
+
+############################################################################
+# Deconvolution functions for consistently accessing each method
+#
+# These functions are called from the generic `deconvolute_mouse()` function.
+# They can also be used by the end-user to access method-specific
+# arguments.
+############################################################################
+
+
+
+#' Deconvolute using mMCP-counter
+#' 
+#' @param gene.expression.matrix a m x n matrix with m genes and n samples. Should be TPM normalized
+#' @param log2 logical. If TRUE, log2 transforms the expression matrix
+#' @export
+#' 
+deconvolute_mMCP = function(gene.expression.matrix, log2 = TRUE){
```

Comment:
for consistency with the rest of the package, I would prefer lower case, `snake_case` function names everywhere.

Reply:
Since we already have the function: 

deconvolute_mcp_counter

I left the other one in uppercase to avoid confusion
Should I change it?